src/legacy/ui/public/courier/fetch/call_client.ts

```diff
@@ -39,15 +48,20 @@ export function callClient(searchRequests, requestsOptions = [], { es, config, e
   Object.keys(searchStrategyMap).forEach(searchStrategyId => {
     const searchStrategy = getSearchStrategyById(searchStrategyId);
     const requests = searchStrategyMap[searchStrategyId];
-    const { searching, abort } = searchStrategy.search({ searchRequests: requests, es, config, esShardTimeout });
+
+    const { searching, abort } = searchStrategy!.search({
```

Comment:
If `getSearchStrategyById` could possibly return `undefined`, then maybe we should be adding an `if (searchStrategy) {}` check here, so that TS doesn't need the `!`

Reply:
Updating `getSearchStrategyById` to never return `undefined`.